modules/host/persist.go

```diff
@@ -172,26 +171,6 @@ func (h *Host) load() error {
 		return err
 	}
 
-	// Get the contract count by observing all of the incomplete storage
-	// obligations in the database.
-	err = h.db.View(func(tx *bolt.Tx) error {
```

Comment:
we could also set `h.financialMetrics.ContractCount = 0` here instead of removing this database scan, not sure which is more correct.